tools/shell/shell.cpp

```diff
@@ -1513,6 +1481,17 @@ void ShellState::OpenDB(ShellOpenFlags flags) {
 			ExecuteQuery("PRAGMA enable_print_progress_bar");
 		}
 	}
+
+	// Register log storage to stdout
+	auto std_out_log_storage = duckdb::make_shared_ptr<ShellLogStorage>(*this);
+	duckdb::shared_ptr<duckdb::LogStorage> storage_ptr = std_out_log_storage;
+
+	auto *db_instance = db->instance.get();
+	auto &log_manager = db_instance->GetLogManager();
+	log_manager.RegisterLogStorage("shell_log_storage", storage_ptr);
+	log_manager.SetLogStorage(*db_instance, "shell_log_storage");
```

Comment:
I'm not 100% sure we trust the logging code enough by now to enable logging by default all the time, perhaps this should be behind some option?